src/query/generated/proto/admin/placement.proto

```diff
@@ -8,6 +8,10 @@ message PlacementInitRequest {
   repeated placementpb.Instance instances = 1;
   int32 num_shards = 2;
   int32 replication_factor = 3;
+
+  // M3Agg placements only
```

Comment:
Maybe we should place these together, i.e.

```proto
message M3AggregatorPlacementOptions {
  uint64 max_aggregation_window_size_nanos = 1;
  uint64 warmup_duration_nanos = 2;
}

message PlacementInitRequest {
  repeated placementpb.Instance instances = 1;
  int32 num_shards = 2;
  int32 replication_factor = 3;
  M3AggregatorPlacementOptions aggregator_options = 4;
}

message PlacementAddRequest {
  repeated placementpb.Instance instances = 1;
  bool force = 2;
  M3AggregatorPlacementOptions aggregator_options = 4;
}
```

Reply:
Any way we can avoid passing `M3AggregatorPlacementOptions` in request? I think it can be configured in the handler

Reply:
The problem is that we can only guess at the correct value. They may not have all the namespaces configured on this coordinator, or the endpoints may be embedded in a different service that doesn't have the info....lets talk offline